0_intro_to_jax_for_ml.ipynb

```diff
@@ -0,0 +1,2081 @@
+{
```

Comment:
<p>Maybe print these numbers, otherwise it feels like running the cell doesn't do anything useful :P </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>

Reply:
<p>I don't want to print these values since they are the answers to next question . </p>